src/game.ts

```diff
@@ -91,36 +68,42 @@ export const marbleWonByPlayer = (graph: Graph): number => {
   return -1;
 };
 
-export const renderScoreBoard = (players: Player[]): void => {
-  console.log("----------------");
-  console.log("Scoreboard");
-  for (const player of players) {
-    let playerScore = `Player ${player.playerNumber} :`;
-    for (const marble of player.marblesWon) {
-      playerScore += marbleValuetoANSIColorCode(marble) + " ";
+export const checkIfPlayerWon = (player: Player) => {
+  let otherPlayerMarbles = 0;
+  let neutralMarbles = 0;
+
+  for (const marble of player.marblesWon) {
+    if (marble !== player.marbleColor) {
+      if (marble === 3) {
+        neutralMarbles++;
+      } else {
+        otherPlayerMarbles++;
+      }
     }
-
-    console.log(playerScore);
   }
-  console.log("----------------");
-};
 
-export const checkIfPlayerWon = (player: Player) => {
-  return player.marblesWon.length === 7;
+  return neutralMarbles === 7 || otherPlayerMarbles === 8;
 };
 
-export const renderWinnerScreen = (player: Player): void => {
-  console.log("\n\n----------------");
-  console.log("----------------");
-
-  console.log(
-    `\n\nCongratulations, the player ${player.playerNumber} has won the Kuba game`
-  );
+export const setGameStateOnDirectionSelected = (
```

Comment:
I think we can find a better name for this function, it does more than what its name tell us... No big deal, but if you find a better name... applyDirection (I try to suggest some ideas :P )?